src/core/zowe/core_for_zowe_sdk/request_handler.py

```diff
@@ -72,6 +72,30 @@ def perform_request(self, method, request_arguments, expected_code=[200]):
         self.__validate_response()
         return self.__normalize_response()
 
+    def perform_streamed_request(self, method, request_arguments, expected_code=[200]):
+        """Execute a streamed HTTP/HTTPS requests from given arguments and return a raw response.
+
+        Parameters
+        ----------
+        method: str
+            The request method that should be used
+        request_arguments: dict
+            The dictionary containing the required arguments for the execution of the request
+        expected_code: int
+            The list containing the acceptable response codes (default is [200])
+
+        Returns
+        -------
+        A raw response data
+        """
+        self.method = method
+        self.request_arguments = request_arguments
+        self.expected_code = expected_code
+        self.__validate_method()
+        self.__send_request(stream=True)
+        self.__validate_response()
+        return self.response.raw
```

Comment:
In order to reach 80% coverage on the Codecov patch check, could you add a unit test for the `perform_streamed_request` method?

There is already a unit test suite for `RequestHandler` where it could be added: https://github.com/zowe/zowe-client-python-sdk/blob/next/tests/unit/test_zowe_core.py#L59-L69

A suggestion for a unit test would be to mock the `__send_request` method (see [here](https://stackoverflow.com/a/47407845/5504760) for an example of mocking a method inside a class), and assert that it is called by `perform_streamed_request`.

Reply:
An issue I face with mocking `send_request` is that it seems because `perform_streamed_request` calls `send_request` it causes an error

Reply:
I think the issue with mocking `__send_request` is that it cannot be mocked because of the double underscores in front which makes it a private method. It may work to mock `requests.Session.send` instead, since it is a public method and mocking it should stop an actual REST request from being made.